ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1838,6 +1838,9 @@ private void LoadConditionalFormatting(ConditionalFormatting conditionalFormatti
                 foreach (var fr in conditionalFormatting.Elements<ConditionalFormattingRule>())
                 {
                     var conditionalFormat = new XLConditionalFormat(ws.Range(sor.Value));
+
+                    conditionalFormat.StopIfTrueInternal = OpenXmlHelper.GetBooleanValueAsBool(fr.StopIfTrue, false);
```

Comment:
This doesn't make sense. OpenXML types, e.g. BooleanValue should not be used here.

Reply:
Can you expand on this? I think it does make sense. We are dealing with a Conditional Format rule from the OOXML lib. Which is returning a BooleanValue, instead of a bool.

![image](https://user-images.githubusercontent.com/22806193/30836491-8bea8722-a22c-11e7-915f-e7b9d3708912.png)

Reply:
Sorry, this time it was my copy and pasting ;-) This change is fine.